tower/src/util/map.rs

```diff
@@ -0,0 +1,61 @@
+use std::task::{Context, Poll};
+use tower_layer::Layer;
+use tower_service::Service;
+
+/// A service to map from one request type to another.
```

Comment:
```suggestion
/// A [`tower::Service`] which maps from one request type to another.
///
/// [`tower::Service`]: [../../src/trait.Service.html]
```

Link might be wrong, though! Render this locally first.